src/azure-cli-core/HISTORY.rst

```diff
@@ -6,6 +6,7 @@ Release History
 ++++++
 * auth: key on both subscription id and name on msi login
 * Add events module in core for EVENT_INVOKER_PRE_CMD_TBL_TRUNCATE
+* Update storage api-version
 * Minor fixes
```

Comment:
You don't need an entry for updating API version. Any if there's literally anything else in the history entry for a release, you don't need "minor fixes".